src/Components/OperationsGroup.jsx

```diff
@@ -35,12 +35,21 @@ export default function OperationsGroup({unSelectItem, installPrefix, fileOpen,
   const isCommentsOn = useStore((state) => state.isCommentsOn)
   const isPropertiesOn = useStore((state) => state.isPropertiesOn)
   const viewer = useStore((state) => state.viewerStore)
+  const cutPlaneDirection = useStore((state) => state.cutPlaneDirection)
   const selectedElement = useStore((state) => state.selectedElement)
   const isMobile = useIsMobile()
   const classes = useStyles({isCommentsOn: isCommentsOn})
   const theme = useContext(ColorModeContext)
 
 
+  const selected = () => {
```

Comment:
can you make this isSelected?  Should follow boolean naming

Reply:
yep